tests/e2e/sentinel/transitions/update_scheduled_reports.spec.js

```diff
@@ -283,6 +283,18 @@ describe('update_scheduled_reports', () => {
       .updateSettings(settings, true)
       .then(() => utils.saveDocs([doc1, doc2, doc3, doc4, doc5]))
       .then(() => sentinelUtils.waitForSentinel([ doc1._id, doc2._id, doc3._id, doc4._id, doc5._id ]))
+      .then(() => utils.getDocs([ doc1._id, doc2._id, doc3._id, doc4._id, doc5._id ]))
```

Comment:
This is a flip around so we check stuff that should happen sync with the transition first, and then reboot sentinel to have infodocs cleaned up and check those